app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :first_name, :last_name, :email, :rents_counter
+  def rents_counter
```

Comment:
This is really good, I think we can try here using the `counter_cache` magic.
Check this out here: http://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache

I will call you later and explain with more details.